apps/prairielearn/assets/scripts/workspaceClient.ts

```diff
@@ -93,13 +99,22 @@ $(function () {
     setMessage(msg.message);
   });
 
+  // Whenever we establish or reestablish a connection, join the workspace room.
   socket.on('connect', () => {
-    socket.emit('joinWorkspace', { workspace_id: workspaceId }, (msg) => {
-      console.log('joinWorkspace, msg =', msg);
-      setState(msg.state);
-    });
+    socket.emit(
+      'joinWorkspace',
+      {
+        token: socketToken,
+        workspace_id: workspaceId,
+      },
```

Comment:
The token is not actually used in `joinWorkspace`, neither will it be used in the follow-up to this PR, so I think it shouldn't be here.

Reply:
Removed in 76df7c0.